src/Engines/nativeEngine.ts

```diff
@@ -279,6 +281,9 @@ export class NativeEngine extends Engine {
     }
 
     public clear(color: Color4, backBuffer: boolean, depth: boolean, stencil: boolean = false): void {
+        if (color == null) { 

```

Comment:
how can color be null?

Reply:
https://github.com/BabylonJS/Babylon.js/blob/master/src/scene.ts#L3916

Reply:
We should maybe do a pass on the method signatures. The main engine has a color signature like this now: `color: Nullable<IColor4Like>`. Doing this will cause TypeScript to give you an error if you don't check for `null`.

Reply:
Also, this check isn't correct. If color is null, you still have to set the backBuffer, depth, and stencil.

Reply:
If we want the same behavior as BJS, yes, we have to.